### PR TITLE
Update documentation

### LINKED ISSUE #189: Document debugging tricks: run gdb in xterm or screen
When you have access to a graphical environment, a nice debugging trick is to do:

```
implementations:
  <component>:
    executable: xterm
    args:
    - -e
    - 'gdb --args <executable> <args...>'
```

This opens a new `xterm` and starts the GNU debugger pointing to your actually executable.

On non-GUI environments a similar thing can be achieved with GNU screen:

```
implementations:
  <component>:
    executable: screen
    args:
    - -dmS
    - gdb_session
    - gdb
    - --args
    - <executable>
    - <arg1>
    - <arg2...>
```

Which starts a detached `screen` session called `gdb_session`. You can attach to this session when executing (in a separate command line):
```
screen -r gdb_session
```

Note: this should not be a detailed explanation on how to use `gdb` and/or `screen`: there's plenty of information on either on the internet.

### PR DESCRIPTION
**Pull request by maarten-ic**
Fixes #189.
Shows how to use docker/singularity containers with muscle (#173)

### CODE REVIEW

**Comment by LourensVeen** [comment]
Nice! Maybe less technically advanced than some of the other stuff, but it's going to be super-useful for the users to have this!

File: docs/source/containers.rst [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+.. code-block:: bash
+
+    $ muscle_manager
+    Usage: muscle_manager [OPTIONS] YMMSL_FILES...
+    [...]
+
+
+.. rubric:: Create containers for the components
+
+For this example we will create a single Singularity container, which contains both the
```
[the remark this answers is not in the document]
Comment (maarten-ic) [reply docs/source/containers.rst:181]: Nice catch!

File: docs/source/containers.rst [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+``muscle3.def`` with the following contents:
+
+.. code-block:: singularity
+
+    Bootstrap: docker
+    From: python:3.10
+
+    %post
+        # download latest muscle3 release from git
+        git clone https://github.com/multiscale/muscle3.git --branch master /opt/muscle3
```
[the remark this answers is not in the document]
Comment (maarten-ic) [reply docs/source/containers.rst:77]: Updated. `containers.rst` is now generated from `containers.rst.in` to allow replacing the version number (same like `installing.rst`).

[the remark this answers is not in the document]
Comment (LourensVeen) [reply docs/source/containers.rst]: Yes, that was the downside, but it's worth it I think. Thanks!

**Review by LourensVeen — APPROVED** [review]
Great

**Pull request merged**